cpp/src/slice2java/Gen.cpp

```diff
@@ -2148,11 +2100,13 @@ Slice::JavaVisitor::writeServantDocComment(Output& out, const OperationPtr& p, c
         out << nl << " * @return A completion stage that the servant will complete when the invocation completes.";
     }
 
-    map<string, StringList> exDocs = dc->exceptions();
-    for (const auto& exDoc : exDocs)
+    for (const auto& [name, lines] : dc->exceptions())
```

Comment:
We have very similar code above, we should consider refactor to a function in a follow-up PR.